src/components/AuthDisplay/AuthDisplay.stories.jsx

```diff
@@ -0,0 +1,21 @@
+// react libraries
+import * as React from 'react';
+import { Provider } from 'react-redux';
+
+// third-party libraries
+import { storiesOf } from '@storybook/react';
+
+import store from 'store/';
+
+// components
+import { AuthDisplay } from './index';
+
+const connectedComponent = (form) => (
+  <Provider store={store}>
+    <AuthDisplay formType={form} />
+  </Provider>
+);
+
+storiesOf('AuthDisplay', module)
+  .add('Auth content proposed in midal', () => connectedComponent('signup'))
```

Comment:
```suggestion
  .add('Auth content proposed in modal', () => connectedComponent('signup'))
```
Please pay more attention to details